apps/openmw/mwmechanics/mechanicsmanagerimp.cpp

```diff
@@ -1355,9 +1355,39 @@ namespace MWMechanics
 
     void MechanicsManager::actorKilled(const MWWorld::Ptr &victim, const MWWorld::Ptr &attacker)
     {
-        if (attacker.isEmpty() || attacker != getPlayer())
+        if (attacker.isEmpty())
             return;
 
+        bool attackerIsPlayerFollowerOrSummon = false;
+        MWWorld::Ptr player = getPlayer();
+
+        if (attacker != player) {
+            // Check if attacker is one of the player followers
+            std::list<MWWorld::Ptr> followers = getActorsSidingWith(player);
+            if (std::find(followers.begin(), followers.end(), attacker) != followers.end())
+                attackerIsPlayerFollowerOrSummon = true;
+
+            // If not, check if attacker is one of the player summoned creatures
+            if (!attackerIsPlayerFollowerOrSummon) {
```

Comment:
If I remove follower code, it should stay here because followers shouldn't raise bounty.